theblues/identity_manager.py

```diff
@@ -61,11 +69,11 @@ def login(self, username, json_document):
     def discharge(self, username, macaroon):
         """Discharge the macarooon for the identity.
 
-        @param username -- the logged in user.
-        @param macaroon -- the macaroon returned from CharmStore.
-
-        Return the resulting base64 encoded macaroon.
         Raise a ServerError if an error occurs in the request process.
+
+        @param username The logged in user.
+        @param macaroon The macaroon returned from the Charmstore.
```

Comment:
I'd use "charm store" everywhere, but that's just personal preference. Anyway, let's use the same spelling everywhere in the code.